help/default/pwsafe.hhp

```diff
@@ -8,7 +8,7 @@ Display compile progress=Yes
 Error log file=pwsafe.log
 Full-text search=Yes
 Index file=pwsafe.hhk
```

Comment:
I'm happy to drop this, however, I did consider that possibility.

I decided it probably wasn't German because there were a couple of other lines that seemed to not be German:
Compatibility=1.1 or later
Language=0x409 English (USA)
Title=HTML Help for Password Safe